.gitattributes

```
@@ -0,0 +1,3 @@
+3rdparty/ linguist-vendored	
+vmlinux.h linguist-generated
+internal/pprof/binutils/testdata/ linguist-vendored
```

```suggestion
internal/pprof/ linguist-vendored
internal/go linguist-vendored
```

I think this is more correct :)